### PR TITLE
Add a debug outline plugin for tailwind

### PR DESCRIPTION
**Pull request by perruche**
Port the debug outline file from scss-toolkit to tailwind-config. 

Debug outlines can be turn on/off using `theme.debugOutline` a Boolean inside tailwind.config.js

### CODE REVIEW

**Comment by perruche** [comment]
> 👍
> 
> Maybe this PR should be made against the `feature/default-config-file` branch to avoid conflict later on.

Yeah that's probably wiser.

File: src/index.js [hunk 1]
```diff
@@ -1,5 +1,9 @@
 module.exports = {
-  theme: {},
+  theme: {
+    debugOutline: true,
```
Comment (perruche) [inline src/index.js]: Should we set default value for `debugOutline` to `false` ?

Comment (titouanmathis) [reply src/index.js]: Yes, that probably is a good idea.

**Comment by titouanmathis** [comment]
@perruche I think we should prefix all the selectors with a `.debug` class or a `data-debug` attribute, so we can quickly toggle the debug when it is enabled.

**Pull request merged**